modules/claims_api/lib/claims_api/common/exceptions/lighthouse/timeout.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module Common
+    module Exceptions
+      module Lighthouse
+        class Timeout < StandardError
+          def initialize(errors)
+            @errors = [errors].flatten
```

Comment:
we do nothing with this currently - is this just providing flexibility for future use cases / expansion?

Reply:
@FonzMP We use it here in the [lighthouse_error_handler](https://github.com/department-of-veterans-affairs/vets-api/blob/API-37794-brd-handle-timeouts/modules/claims_api/lib/claims_api/v2/error/lighthouse_error_handler.rb#L29)

Reply:
I'm meaning line 9. 
`@errors = [errors].flatten`

we don't use @errors again, right?